### ISSUE TITLE
Problem decoding tuple of structs involving strings

### ISSUE THREAD

**Issue by JackieWeiss** [status-im/nim-ethers#18]
The following 2 examples work as expected
```solidity
//works
contract Test{
    struct FooStruct{
        uint256 somevalue1;
        uint256 somevalue2;
    }
    function getfoo() external view returns(FooStruct memory, FooStruct memory){
        return (FooStruct(1, 1), FooStruct(2, 2));
    }
}
```
```solidity
//works
contract Test{
    struct FooStruct{
        string teststring1;
        uint256 somevalue1;
    }
    function getfoo() external view returns(FooStruct memory){
        return FooStruct("1", 1);
    }
}
```
But returning 2 structs with strings
```solidity
//broken
contract Test{
    struct FooStruct{
        string teststring1;
        uint256 somevalue1;
    }
    function getfoo() external view returns(FooStruct memory, FooStruct memory){
        return (FooStruct("1", 1), FooStruct("2", 2));
    }
}
```
fails with error
`Error: unhandled exception: unable to decode return value as (FooStruct, FooStruct) [ContractError]`

**Comment by markspanbroek** [comment]
Interesting. Thanks for testing, I'll take a look.

**Comment by markspanbroek** [comment]
Fixed in version 0.1.9.
